doc/stages/readers.stac.rst

```diff
@@ -20,8 +20,8 @@ Example
             "type": "readers.stac",
             "filename": "https://s3-us-west-2.amazonaws.com/usgs-lidar-stac/ept/catalog.json",
             "reader_args": [{"type": "readers.ept", "resolution": 100}],
-            "item_ids": ["MD_GoldenBeach_2012"],
```

Comment:
Do you have synonyms for this change?

Reply:
I don't, should there be synonyms? `catalog_ids` and `collection_ids` just started feeling too long so I removed the `ids` from them all

Reply:
Ahhh, makes sense. Okay,  I'll add them.